packages/contracts-bedrock/addresses.json

```diff
@@ -0,0 +1,253 @@
+{
+  "LegacyMessagePasser": {
```

Comment:
Why does this file exist in 2 packages?

Reply:
It exists in contracts-bedrock because that's where it belongs
It exists in contracts-ts so it's there for folks installing contracts-ts also have an easy to reference json file